### PR TITLE
[fix](Nereids) fix `''` and `""` in string literal

### PR DESCRIPTION
**Pull request by keanji-x**
## Proposed changes

`''` represents `'` in sql standard
`""` represents `"` in sql standard

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java [hunk 1]
```diff
@@ -326,19 +327,7 @@ public Cost visitPhysicalHashJoin(
...
-                    || (probe.columnStatistics().containsKey(slot) && !probe.columnStatistics().get(slot).isUnKnown)) {
-                continue;
-            }
-            return true;
-        }
-        return false;
-    }
-
-    private boolean isStatsUnknown(PhysicalNestedLoopJoin<? extends Plan, ? extends Plan> join,
+    private boolean isStatsUnknown(AbstractPhysicalJoin<? extends Plan, ? extends Plan> join,
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java:330]: move these refactors to another pr

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by jackwener — APPROVED** [review]
